### ISSUE TITLE
Midpoint Review - Design Feedback - IVC Forms, 10-10d

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/va.gov-team#76371]
## Next Steps for the VFS team

- [x] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- There are a lot of alerts and we do know that VA has alert fatigue. You may want to evaluate each alert and decide wether or not it is necessary.
- Test around the 2 file upload options, I am not sure it is necessary and it does extend the form quite a bit. We currently do not do this in any other form and we have not heard that it is an issue.
- I did some digging around our discussion for uploading multiple files on a single page. If the content is different for each file then yes, you are correct and should use the one thing per page mentality. If the content is mostly the same then you may be able to utilize the pattern where can [upload another file](https://design.va.gov/patterns/ask-users-for/files#review).
- Matt: Make sure you have a design that handles this status around missing uploaded files and allow them to finish online if they chose to do so. And we do not have great patterns for this. Some forms handle it with an alert. Definitely research the flow.
- Matt: In terms of questions around housing and homelessness reach out to the CAIA team. We do not have a pattern on how we do this in other forms. You could use the sub task pattern of asking about homelessness.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] Must: Add a "REMOVE" button to ~multiple responses~ applicant review card.
- <img width="174" alt="Screenshot 2024-02-16 at 10 34 58 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/72398073/fcc56458-40c4-47e5-9745-e4b7b55f69b5">
- [ ] Must: Make the edit link on the ~multiple responses~ applicant review card an [active link style](https://design.va.gov/components/link/#active).
- [ ] Must: Use the UI from the [slim alert v3](https://design.va.gov/components/alert#examples---slim-alert---v3) on the review page.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [x] Add the Collaboration Cycle initiative epic

**Comment by syd-hoeper** [comment]
@allison0034 Hi there, I was wondering if we could get a little more information around the the feedback regarding multiple responses? I'm looking at the Multiple Responses pattern ([https://design.va.gov/patterns/ask-users-for/multiple-responses#how-to-design-and-build---multi-page](https://design.va.gov/patterns/ask-users-for/multiple-responses#how-to-design-and-build---multi-page)) and not seeing those red "remove" buttons. I'm wondering if I could get the link to where that screenshot was taken? I'm also wondering if you could explain which link in the multiple responses needs to be in the active link style? It looks like we're using the patterns stated in that link I just posted, so just looking for some clarity on the specifics. Thank you so much!

**Comment by syd-hoeper** [comment]
@humancompanion-usds Regarding the slim alert being used on the Review page, I'm wondering: do you mean it should be used in place of the standard alert when a user still has items to mail in? Or is the slim alert usage referring to when a user makes an update to the page that requires a new file? I'm asking because I was under the impression that a slim alert is only to be used when a user takes an action on the page and there's an associated update ([see this Slack thread](https://dsva.slack.com/archives/C01DBGX4P45/p1706134328428459?thread_ts=1706134017.305679&cid=C01DBGX4P45) so I can definitely understand the second scenario I laid out, but just wanted to clarify to see if the feedback was about the standard alert currently in there.
Thank you!

**Comment by syd-hoeper** [comment]
@humancompanion-usds Thank you so much, that's super helpful info!

**Comment by allison0034** [comment]
Hi @syd-hoeper , I apologize, the remove needs to be added to the applicant review screen. I am getting some clarification from Matt Dingee on the exact specs as it may have changed recently.
<img width="246" alt="Screenshot 2024-02-15 at 2 02 10 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/72398073/f8627b33-b42d-4d25-8ed4-47ec37f75a37">

**Comment by syd-hoeper** [comment]
@allison0034 @humancompanion-usds Hi again, so the issue is actually that we forgot to remove that screen from the Figma -- that was a recreation of Jeanna's new list and loop pattern, and when we updated the screens to match the existing list and loop that we're going to use, this screen never got removed. So I'm going to get rid of this page in our Figma and we won't test with it in.

That being said, I talked to our engineer and if the user testing shows that people are too confused, we might be able to custom code a review screen that matches the new list and loop pattern until that pattern is actually ready for us use.

Thank you both so much for the feedback!
